src/WCSimPMTObject.cc

```diff
@@ -1343,13 +1351,17 @@ G4float* BoxandLine20inchHQE::GetQEWavelength(){
 }
 
 G4float* BoxandLine20inchHQE::GetQE(){
+  G4float correctionFactor = 1./0.73;//Correction factor added in July 2015 to scale the output of B&L PDs to 1.9 times the 20" PMTS based on Hamamatsu measurement. This was done in Pull Request #98 and will be removed once a more permanent solution is found.
+  // set R3600 C.E. to 73% and not to change the R3600 output.
   static G4float QE[20] =
-    { 0.00, .0008, .1255, .254962, .2930, .3127, .3130, .2994, .2791, .2491,
-      .2070,  .1758, .1384, .0779, .0473, .0288, .0149, .0062, .0002, .0001};  
+    { 0.00*correctionFactor, .0008*correctionFactor, .1255*correctionFactor, .254962*correctionFactor, .2930*correctionFactor, .3127*correctionFactor, .3130*correctionFactor, .2994*correctionFactor, .2791*correctionFactor, .2491*correctionFactor,
+      .2070*correctionFactor,  .1758*correctionFactor, .1384*correctionFactor, .0779*correctionFactor, .0473*correctionFactor, .0288*correctionFactor, .0149*correctionFactor, .0062*correctionFactor, .0002*correctionFactor, .0001*correctionFactor};  
+
   return QE;
 }
 G4float BoxandLine20inchHQE::GetmaxQE(){
-  const G4float maxQE = 0.315;
+  G4float correctionFactor = 1./0.73;//Correction factor added in July 2015 to set R3600 C.E. to 73% and not to change the R3600 output.
```

Comment:
Remove this comment. It doesn't apply now that we aren't changing R3600 to 73%. Replace it with the comment on line 1354 //Correction factor added in July 2015 to scale the output of B&L PDs to 1.9 times the 20" PMTS based on Hamamatsu measurement. This was done in Pull Request #98 and will be removed once a more permanent solution is found.